perlop: Update extra paired delims to Unicode 16

khwilliamson opened the pull request:
One new pair is added since Unicode 14.

The verbatim indentation is lowered so some items that required two lines now take one.


* This set of changes does not require a perldelta entry.  This is merely updating the documentation to reflect what actually happ0ens

jkeenan commented:
> One new pair is added since Unicode 14.
> 

Which pair is that?

khwilliamson commented:
The final one on the page.  I didn't realize that github didn't do a good job of showing showing which things are white space only.  I'm using the "refined github" add-on.  So, I don't know if this is available in plain github or not, but it has the option to display diffs in rich format.  Doing so cut the displayed differences way down.

I could split the commit into two, the first being white-space only.

Leont commented:
> I could split the commit into two, the first being white-space only.

Yeah, I think that's always a good idea.

khwilliamson committed:
perlop: Whhite space + rmv =cut line

The verbatim indentation is lowered so some items that required two
lines now take one.

The =cut line is not needed in a pod file; it adds nothing here, so is
removed

khwilliamson committed:
perlop: Update extra paired delims to Unicode 16

One new pair is added since Unicode 14.
